ui/nuxeo-filter.js

```diff
@@ -178,12 +180,38 @@ import { FiltersBehavior } from './nuxeo-filters-behavior.js';
     }
 
     _evaluate(document, user, expression) {
+      let res = false;
+
       try {
-        const fn = new Function(['document', 'user'], `return ${expression};`);
-        return fn.apply(this, [document, user]);
+        if (Nuxeo.UI && Nuxeo.UI.config && Nuxeo.UI.config.expressions && Nuxeo.UI.config.expressions.eval === false) {
```

Comment:
I confess I don't recall it correctly, but I think I had some issues in the past because all this configs are strings.
This to say that `=== false` may not work 🤔 

Anyway, confirmation is needed 😅

Reply:
that is a very good point! thanks 👍

Reply:
You're welcome! 🙂

Reply:
Fixed 👍